libgreat/host/pygreat/comms.py

```diff
@@ -540,6 +546,16 @@ def execute_command(self, class_number, verb, in_format, out_format, *arguments,
         Returns any data recieved in response, parsed according to out_format.
         """
 
+        # Emulate python3 keyword-only arguments.
+        timeout  = kwargs.pop('timeout', 1000)
+        encoding = kwargs.pop('encoding', None)
+        max_response_length = kwargs.pop('max_response_length', 4096)
+        comms_timeout = kwargs.pop('comms_timeout', 1000)
+        name = kwargs.pop('name', "anonymous")
+
+        if kwargs:
+            raise TypeError("Unexpected keyword arguments: {}".format(kwargs.values()))
```

Comment:
This should probably use `kwargs.keys()`.

Reply:
Yep; this was a genuine typo. Thanks.